### PR TITLE
Hovering Effect Added in explore section on More details button #249

### PR DESCRIPTION
**Pull request by ishitakapoor26**
## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

**Comment by ishitakapoor26** [comment]
Pls merge this pr for issue#249 and close it @Praful932 @maneprajakta

**Comment by maneprajakta** [comment]
@ishitakapoor26 don't change the hover effect for other buttons.
Just add `text-shadow to the `more-details` button.

**Comment by ishitakapoor26** [comment]
> @ishitakapoor26 don't change the hover effect for other buttons.
> Just add `text-shadow` to the `more-details` button.

ok

**Comment by ishitakapoor26** [comment]
@maneprajakta added hover effect to only more details button and removed hover effect from rest.

File: static/mainapp/css/read.css [hunk 1]
```diff
@@ -92,8 +95,7 @@
   }
   
   .rating-block:hover {
-    text-shadow: 0px 0px 6px rgba(255, 255, 255, 1);
-    transition: all 0.4s ease 0s;
+    color: white;
```
Comment (maneprajakta) [inline static/mainapp/css/read.css:98]: keep it same as it was earlier

File: static/mainapp/css/layout.css [hunk 2]
```diff
@@ -228,7 +228,7 @@ footer a:hover {
     margin-top: -25px;
   }
   .nav-link:hover {
-    text-decoration: underline;
+      text-shadow: 2px 2px 6px white;
```
Comment (maneprajakta) [inline static/mainapp/css/layout.css:231]: keep it same as earlier one

File: static/mainapp/css/index.css [hunk 3]
```diff
@@ -321,8 +324,7 @@ ol.carousel-indicators li.active {
 }
 
 .rating-block:hover {
-  text-shadow: 0px 0px 6px rgba(255, 255, 255, 1);
-  transition: all 0.4s ease 0s;
+  color: white;
```
Comment (maneprajakta) [inline static/mainapp/css/index.css:327]: Don't change this

File: static/mainapp/css/genre.css [hunk 4]
```diff
@@ -111,8 +114,7 @@
 }
 
 .rating-block:hover {
-  text-shadow: 0px 0px 6px rgba(255, 255, 255, 1);
-  transition: all 0.4s ease 0s;
+  color: white;
```
Comment (maneprajakta) [inline static/mainapp/css/genre.css:117]: Don't change this

File: static/mainapp/css/explore.css [hunk 5]
```diff
@@ -162,9 +165,9 @@
   padding: 8px;
 }
 
-.rating-block:hover {
-  text-shadow: 0px 0px 6px rgba(255, 255, 255, 1);
-  transition: all 0.4s ease 0s;
+.rating-block.btn:hover {
+  color: white;
```
Comment (maneprajakta) [inline static/mainapp/css/explore.css:169]: don't change this

File: readme.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```
Comment (maneprajakta) [inline readme.html:1]: remove the entire html file

File: LICENSE [hunk 7]
```diff
@@ -1,13 +1,11 @@
-MIT License
-
-Copyright (c) 2020 Praful932
+Copyright (c) 2017 Prasanth Janardanan
```
Comment (maneprajakta) [inline LICENSE:1]: remove all the changes from this.

**Review by maneprajakta — CHANGES_REQUESTED** [review]

**Review by maneprajakta — COMMENTED** [review]
You just need to change CSS for the `more-details` button don't change CSS for other buttons.

**Comment by Praful932** [comment]
@ishitakapoor26 It would be better if you keep the PR conversation in the Pull request chat :)
Your recent commit is on a different branch(master) and your PR is on a different branch(Ishita)
Please check

**Pull request closed without merging**